frontend/appflowy_flutter/lib/plugins/database/widgets/cell/mobile_grid/mobile_grid_date_cell.dart

```diff
@@ -25,9 +27,17 @@ class MobileGridDateCellSkin extends IEditableDateCellSkin {
         child: SingleChildScrollView(
           scrollDirection: Axis.horizontal,
           padding: const EdgeInsets.symmetric(horizontal: 12, vertical: 10),
-          child: FlowyText(
-            state.dateStr,
-            fontSize: 15,
+          child: Row(
+            children: [
+              if (state.data?.reminderId.isNotEmpty ?? false) ...[
+                const FlowySvg(FlowySvgs.clock_alarm_s),
+                const HSpace(6),
+              ],
+              FlowyText(
+                state.dateStr,
+                fontSize: 15,
```

Comment:
I missed handling the overflow here. Can you fix it for me?

Reply:
Will do

Reply:
Oh nvm. It's in a singlechildscrollview

Reply:
Even better